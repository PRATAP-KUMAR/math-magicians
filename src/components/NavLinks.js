import { Link } from 'react-router-dom';
import './navlinks.css';

const NavLinks = () => (
  <>
    <header className="header">
      <h1>Math Magicians</h1>
      <nav>
        <ul>
          <li><Link to="/">Home</Link></li>
          <li><Link to="/Calculator">Calculator</Link></li>
          <li><Link to="/Quote">Quote</Link></li>
        </ul>
      </nav>
    </header>
  </>
);

export default NavLinks;
