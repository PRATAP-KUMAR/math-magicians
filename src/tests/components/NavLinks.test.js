import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';

import NavLinks from '../../components/NavLinks';

describe('NavLinks Test', () => {
  const { container } = render(<MemoryRouter><NavLinks /></MemoryRouter>);
  test('test for NavLinks', () => {
    expect(container.firstChild.className).toBe('header');
    expect(container.firstChild).toMatchSnapshot();
  });
});
