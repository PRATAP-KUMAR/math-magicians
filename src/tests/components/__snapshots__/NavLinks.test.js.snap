// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLinks Test test for NavLinks 1`] = `
<header
  class="header"
>
  <h1>
    Math Magicians
  </h1>
  <nav>
    <ul>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/Calculator"
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/Quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
